intlogger.go

```diff
@@ -353,32 +375,69 @@ func (l *intLogger) logPlain(t time.Time, name string, level Level, msg string,
 				}
 			}
 
-			var key string
-
+			// Convert the field key to a string.
 			switch st := args[i].(type) {
 			case string:
 				key = st
 			default:
 				key = fmt.Sprintf("%s", st)
 			}
 
-			if strings.Contains(val, "\n") {
-				l.writer.WriteString("\n  ")
-				l.writer.WriteString(key)
-				l.writer.WriteString("=\n")
-				writeIndent(l.writer, val, "  | ")
-				l.writer.WriteString("  ")
-			} else if !raw && needsQuoting(val) {
+			// Optionally apply the ANSI "dim" (faint) and "bold"
+			// SGR values to the key.
+			if l.fieldColor != ColorOff {
+				color := color.New(color.Faint, color.Bold)
```

Comment:
It seems weird to use faint and bold. Why not just bold?

Reply:
I feel like faint helps break up a log line visually, for me, better than just bold.

### Just Bold

<img width="565" alt="Screen Shot 2022-08-29 at 1 31 52 PM" src="https://user-images.githubusercontent.com/14850816/187262675-d1d9f111-9115-40da-baf7-cce31f8231e4.png">

### Bold and Faint

<img width="566" alt="Screen Shot 2022-08-29 at 1 32 25 PM" src="https://user-images.githubusercontent.com/14850816/187262758-105dd152-d9dd-4e67-88e2-d379b4ec6394.png">

Reply:
The concern is mostly how other folks default terminal fonts will handle faint+bold. We have to be careful picking here because it has a big influence on all the users.

Reply:
That's a fair point. I think the Windows terminal would _likely_ be the odd one to consider, and it seems like they do support this style: https://github.com/microsoft/terminal/issues/6703#issuecomment-678465981

Faint/dim is a fairly standard SGR style, implemented in VSCode's terminal emulator ([xtermjs](https://xtermjs.org/docs/api/vtfeatures/)) as well (which is what those screenshots are showing).

Bold, I believe, is the more widely supported style. But, I would really rather have faint than bold here to help break up the line. I feel it's important. I don't know if we can actually detect if it's supported?